File: labs/OOP/Lab_06/Lab_6/Symbols.cpp
#include "Symbols.h"


int Symbols::RowLength() {
    length = strlen(row);
    return length;

 }
char* Symbols::ChangeSymbol()
{
    char y = '#';
    char x = '!';
    for (int i = 0; i < strlen(row); i++)
    {
        if (row[i] == y) {
            row[i] = x;
        }
    }
    return row;
}

File: labs/OOP/Lab_06/RowsLibrary/Symbols.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace RowsLibrary
{
    public class Symbols : Rows
    {
        public Symbols(string row) : base(row)
        {
        }
        public override int RowLength()
        {
            length = row.Length;
            return length;
        }
        public override string ChangeSymbol()
        {
            row = row.Replace("#", "!!");
            return row;
        }
    }
}
